src/app/modules/entities/state-entity-list/state-entity-list.component.ts

```diff
@@ -0,0 +1,99 @@
+import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
+import { MatPaginator, MatTableDataSource, PageEvent, MatDialog } from '@angular/material';
+import { EntityService } from '../../admin-core/services/entity-service/entity.service';
+import { CommonServiceService } from '../../admin-core';
+import { Router } from '@angular/router';
+
+
+@Component({
+  selector: 'app-state-entity-list',
+  templateUrl: './state-entity-list.component.html',
+  styleUrls: ['./state-entity-list.component.scss']
+})
+export class StateEntityListComponent implements OnInit {
+  EntityStateList: any;
+  dataSource: MatTableDataSource<any>;
+  displayedColumns: any = [];
+  listing: boolean = false;
+  recordCount: any;
+  columns: any;
+  fieldsForOrganisation: any;
+  formdata: any;
+  confirmPopupResult: any;
+  status: any = '';
+  orgObject: any;
+  assignedStatus: any;
+  search: any;
+  noData: boolean;
+  queryParams = {
+    page: 1,
+    size: 10,
+  };
+  @ViewChild('searchInput') searchInput: ElementRef;
+  @ViewChild(MatPaginator) paginator: MatPaginator;
+
+  constructor(private entityService: EntityService,
+    private commonServiceService: CommonServiceService,
+    private router: Router) { }
+
+  ngOnInit() {
+    this.getEntityStateList();
+  }
+
+  applyFilter(event: Event) {
+    const filterValue = (event.target as HTMLInputElement).value;
```

Comment:
Why is this HTMLInputElement

Reply:
To get the value of the input field, catching through event

Reply:
We don't need HTMLInputElement to get the value of field please check

Reply:
Removed